packages/business-rules/src/lib/pins-yup.js

```diff
@@ -2,7 +2,6 @@ const yup = require('yup');
 const { appeal } = require('../validators');
 
 yup.addMethod(yup.date, 'isInThePast', appeal.decisionDate.isInThePast);
-yup.addMethod(yup.date, 'isWithinDeadlinePeriod', appeal.decisionDate.isWithinDeadlinePeriod);
```

Comment:
By defining `isWithinDeadlinePeriod` here it will be available as a method of `pinsYup` object to be used for validation without needing to import it again.

Does it need to b removed here?

Reply:
Reinstated